### ISSUE TITLE
Setting up Command Line Interface

### ISSUE THREAD

**Issue by Noah-Kurz** [amosproj/amos2022ss07-nft-playbook#7]
## User story
1. As a power user
2. I want / need a CLI 
3. So that I can interact with the tool

## Acceptance criteria
* program starts in a shell
* can parse input
* document all dependencies

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release

actual size = 3
